packages/transmute-did/src/lib/openpgpExtensions/did/ocap/json-ld/__tests__/openpgp-ocap-ld.js

```diff
@@ -0,0 +1,86 @@
+const base64url = require('base64url');
+
```

Comment:
name of the file should finish by .spec.js

Reply:
👍